lambdas/ecs-events-to-slack/notify.py

```
@@ -0,0 +1,142 @@
+                        "value": env_vars_formatted,
+                        "short": "true"
+                    },
+                    {
+                        "title": "Networking Status",
+                        "value": attachment_status,
+                        "short": "true"
+                    }
+                ],
+                "actions": [
+                    {
+                      "type": "button",
+                      "text": "Logs",
+                      "url": logs_url
+                    }
+                ],
+                "color": message_color,
+                "footer": "Event id: %s (version: %s at %s)"
+                % (event_id, version, formatted_time),
+            }
+        ]
+    }
+
+    return attachment
+
+
+def get_secret(secret_name, region):
+    session = boto3.session.Session()
+    client = session.client(
+        service_name="secretsmanager",
+        region_name=region,
+    )
+
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        if e.response["Error"]["Code"] == "DecryptionFailureException":
+            print("""Secrets Manager can't decrypt the protected secret text
+            using the provided KMS key:""", e)
+            raise e
+        elif e.response["Error"]["Code"] == "InternalServiceErrorException":
+            print("An error occurred on the server:", e)
+            raise e
+        elif e.response["Error"]["Code"] == "InvalidParameterException":
+            print("The request had invalid params:", e)
+            raise e
+        elif e.response["Error"]["Code"] == "InvalidRequestException":
+            print("The request was invalid due to:", e)
+            raise e
+        elif e.response["Error"]["Code"] == "ResourceNotFoundException":
+            print("The requested secret " + secret_name + " was not found")
+            raise e
```

Soooo I think if you get any other type of ClientError error code, it'll just return nothing - do you want to raise an exception in a generic else statement?